pages/blog/2022-summary.md

```diff
@@ -0,0 +1,252 @@
+
+<Figure
+  src="/img/posts/2022-summary/github-issues.webp"
+/>
+
+Three years ago, we dreamed about having thousands of contributions. We forgot to balance that number with the proper number of maintainers. Now we need to react and make sure we also get a large number of active maintainers.
+
+## npm
+
+Most of our tools are written in JavaScript/TypeScript; we host them on Node Package Manager (npm).
+
+### Top packages
+
+Our most essential packages are [the project that provides AsyncAPI JSON Schema files](https://www.npmjs.com/package/@asyncapi/specs), [the Generator](https://www.npmjs.com/package/@asyncapi/generator), [the React component](https://github.com/asyncapi/asyncapi-react) and [the Parser](https://www.npmjs.com/package/@asyncapi/parser). Below you can see how downloads of these packages increased.
+
+<Figure
+  src="/img/posts/2022-summary/npm-stats.webp"
+/>
+
+You do not believe in this data? Lemme repeat what I wrote last year:
+
+> The increase in numbers is insanely huge. People automatically say that it is because of build systems, local development, or some single corporation started using it at scale. In other words, the increase could result from just one project that uses these tools as a dependency.
+<br/>Let me challenge you:<br/>
+- Corporations always use mirrored versions in npm. When you work on a project in a corporate network, you will always hit internal proxy. Not included in downloads.<br/>
+- Most build systems cache packages that are downloaded already to improve performance. Not included in downloads.<br/>
+- When you work with npm locally, you also have a local cache that is used if you want to install something that you already installed. Not included in downloads.
+
+An interesting observation is that in 2021 AsyncAPI Parser was the only user of the AsyncAPI spec JSON schemas project (`@asyncapi/specs`) as the number of downloads was almost the same. In 2022 AsyncAPI spec JSON schemas project was downloaded five million times, three more than the parser. This means many more tools are being dependent on the project.
+
+In 2021 we could see AsyncAPI Parser is not only Generator's dependency; now we see AsyncAPI spec JSON schemas project is not only Parser's dependency. This means that the increase in traffic in different channels and the growing user base also reflect the number of tools being built on top of AsyncAPI tools. One of big users of `@asyncapi/specs` is now Spectral. Also GitHub tells us that `@asyncapi/specs` is used in 1 329 public repositories 
+
+## YouTube
+
+The [AsyncAPI YouTube channel](https://www.youtube.com/asyncapi) had a massive increase in traffic. 
+
+### Watch time
+
+YouTube is just another channel where we can see the audience is growing. The conference and other regular live streams bring a lot of watchers. In 2023 numbers might drop because we do not want to do the online conference, but yeah, let's see.
```

Comment:
```suggestion
YouTube is just another channel where we can see the audience is growing. The conference and other regular live streams bring a lot of watchers. In 2023, numbers might drop because we do not want to do the online conference, but yeah, let's see.
```